### ISSUE TITLE
Issue after update to phoenix 0.14

### ISSUE THREAD

**Issue by janjiss** [bitwalker/exrm#149]
```
Tasks: TOP => phoenix:release
(See full trace by running task with --trace)
The deploy has failed with an error: Exception while executing as root@elixirstream.com: mix exit status: 1
mix stdout: ==> Building release with MIX_ENV=prod.
==> Generating relx configuration...
==> Generating sys.config...
==> Generating boot script...
==> Conform: Loading schema...
==> Conform: No schema found, conform will not be packaged in this release!
==> Performing protocol consolidation...
==> Generating release...
==> Generated .appup for elixir_stream 0.0.1 -> 0.0.2
==> Failed to build release. Please fix any errors and try again.
mix stderr: Nothing written
```
I like the error message :(

Will try to debug this myself as well.

**Comment by janjiss** [comment]
Ah, okay, I needed to know how to debug this. Ill  do it as soon as I have a chance

**Comment by janjiss** [comment]
Okay, here is the error that I am having: https://gist.github.com/janjiss/4254a41a6b44c6cff76b

**Comment by janjiss** [comment]
Okay, don't know if it's because I am doing thing weirdly or what, but when I bumped the version, everything went well. I use Capistrano to deploy the app and it seems that there is something bad going on if I update dependencies and don't bump up the version of the app.

**Comment by bitwalker** [comment]
@janjiss In order to use upgrades, appups have to be present for any applications which have changed, something recently pointed out to me by @tsloughter. I would've expected it to do the same thing when you bumped the version of your app, but it didn't, so I'll have to look a little closer at that to see why that may have been.

**Comment by bitwalker** [comment]
@janjiss This is now fixed in master, and will be on hex soon as 0.19.0. exrm will generate appups for dependencies when they do not exist. If they do, then exrm will do nothing, but at least this should work as a decent compromise.
